content/index/index.html

```diff
@@ -474,441 +474,101 @@ <h2 id="examples_by_props_label">Examples By Properties and States</h2>
-                <li><a href="../patterns/slider/examples/slider-seek.html">Media Seek Slider</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
-                <li><a href="../patterns/slider/examples/slider-temperature.html">Vertical Temperature Slider</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
-                <li><a href="../patterns/spinbutton/examples/datepicker-spinbuttons.html">Date Picker Spin Button</a></li>
-                <li><a href="../patterns/toolbar/examples/toolbar.html">Toolbar</a></li>
-              </ul>
-            </td>
-          </tr>
-          <tr>
-            <td><code>aria-valuetext</code></td>
-            <td>
-              <ul>
-                <li><a href="../patterns/slider/examples/slider-rating.html">Rating Slider</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
-                <li><a href="../patterns/slider/examples/slider-seek.html">Media Seek Slider</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
-                <li><a href="../patterns/slider/examples/slider-temperature.html">Vertical Temperature Slider</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
-                <li><a href="../patterns/spinbutton/examples/datepicker-spinbuttons.html">Date Picker Spin Button</a></li>
-                <li><a href="../patterns/toolbar/examples/toolbar.html">Toolbar</a></li>
+                <li><a href="../patterns/alertdialog/examples/alertdialog.html">アラートダイアログの例</a></li>
+                <li><a href="../patterns/combobox/examples/combobox-datepicker.html">日付ピッカーコンボボックスの例</a> (<abbr title="ハイコントラストサポート">HC</abbr>)</li>
+                <li><a href="../patterns/dialog-modal/examples/datepicker-dialog.html">日付ピッカーダイアログの例</a> (<abbr title="ハイコントラストサポート">HC</abbr>)</li>
+                <li><a href="../patterns/dialog-modal/examples/dialog.html">モーダルダイアログの例</a></li>
+                <li><a href="../patterns/feed/examples/feed.html">無限スクロールフィードの例</a></li>
+                <li><a href="../patterns/table/examples/table.html">テーブルの例</a></li>
               </ul>
             </td>
           </tr></tbody>
       </table>
     </section>
     <section id="examples_experimental">
-      <h2 id="examples_experimental_label">Experimental Examples</h2>
-      <div><strong>NOTE:</strong> The HC abbreviation means example has High Contrast support.</div>
+      <h2 id="examples_experimental_label">実験的な例</h2>
+      <div><strong>注記:</strong> HC の略語は、例にハイコントラストのサポートがあることを意味します。</div>
       <ul id="examples_experimental_ul">
-                <li><a href="../patterns/tabs/examples/tabs-actions.html">Experimental  Tabs with Action Buttons</a> (<abbr title="High Contrast Support">HC</abbr>)</li></ul>
+                <li><a href="../patterns/tabs/examples/tabs-actions.html">実験的アクションボタン付きタブの例</a> (<abbr title="ハイコントラストサポート">HC</abbr>)</li></ul>
```

Comment:
遷移先のページタイトルは「Experimental Example of Tabs with Action Buttons」なので、「実験的」は「例」に係るのが良いかもしれません。

```suggestion
                <li><a href="../patterns/tabs/examples/tabs-actions.html">アクションボタン付きタブの実験的な例</a> (<abbr title="ハイコントラストサポート">HC</abbr>)</li></ul>
```